app/connectors/httpParsers/ErrorResponse.scala

```diff
@@ -23,8 +23,6 @@ trait ErrorResponse {
   val body: String
 }
 
-case class UnexpectedFailure(override val status: Int, override val body: String) extends ErrorResponse
```

Comment:
Something tiny but could the test description be changed to match the DefaultedUnexpectedFailure used
https://github.com/hmrc/charities-registration-frontend/blob/d45cc2b82f153c07c364b3bb53d8e6322decd850/test/connectors/httpParsers/AddressLookupInitializationHttpParserSpec.scala#L59C8-L59C8